src/main/java/mate/controller/car/CreateCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Manufacturer;
+import mate.service.CarService;
+import mate.service.ManufacturerService;
+
+@WebServlet(urlPatterns = "/car/add")
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-web/web-practice/java-web-practice#:~:text=Good%20URLs%20naming%20in%20your%20project%3A
the same for other urls